src/main/java/org/sagebionetworks/template/repo/RepositoryTemplateBuilderImpl.java

```diff
@@ -388,4 +408,19 @@ String extractDatabaseSuffix(Stack sharedResouces) {
 		throw new RuntimeException("Failed to find shared resources output: "+outputName);
 	}
 
+	/***
+	 * Return the private subnet ids for a color
+	 * @param color
+	 * @return
+	 */
+	List<String> getPrivateSubnets(String color) {
+		String stack = config.getProperty(PROPERTY_KEY_STACK);
+		String privateSubnets = cloudFormationClient.getOutput(
+				Constants.createVpcPrivateSubnetsStackName(stack, color),
+				Constants.VPC_PRIVATE_SUBNETS_STACK_PRIVATE_SUBNETS_OUPUT_KEY);
+		String[] privateSubnetIds = privateSubnets.split(",");
```

Comment:
Even though it's an edge case, if we have a single subnet this will return an empty array